test/ask/runtime/channel_broker_state_test.exs

```diff
@@ -69,9 +69,9 @@ defmodule Ask.Runtime.ChannelBrokerStateTest do
       mock_time(now)
 
       State.new(0, "sms", %{})
-      |> State.queue_contact({%{id: 2, disposition: :queued}, "secret", []}, 5)
+      |> State.queue_contact({%{id: 2, disposition: :queued}, "secret", [], nil, nil}, 5)
```

Comment:
Shall we set non-nil times here, so we don't add coverage for a case we don't want to support?